lib/raven/integrations/rails/active_job.rb

```diff
@@ -1,31 +1,53 @@
 module Raven
   class Rails
-    module ActiveJob
+    module ActiveJobExtensions
+      ALREADY_SUPPORTED_SENTRY_ADAPTERS = %w(
+        ActiveJob::QueueAdapters::SidekiqAdapter
+        ActiveJob::QueueAdapters::DelayedJobAdapter
+      ).freeze
+
       def self.included(base)
         base.class_eval do
-          rescue_from(Exception) do |exception|
-            # Do not capture exceptions when using Sidekiq so we don't capture
-            # The same exception twice.
-            unless self.class.queue_adapter.to_s == 'ActiveJob::QueueAdapters::SidekiqAdapter'
-              active_job_details = {
-                :active_job => self.class.name,
-                :arguments => arguments,
-                :scheduled_at => scheduled_at,
-                :job_id => job_id,
-                :locale => locale
-              }
+          around_perform do |job, block|
```

Comment:
@nateberkopec I was browsing source code and found that this change can _potentially_ lead to Sentry ignoring errors because ActiveJob callbacks are not chained.
[Calling the same callback multiple times will overwrite previous callback definitions](https://github.com/rails/rails/blob/v5.2.3/activejob/lib/active_job/callbacks.rb#L18)

Reply:
Never mind. Everything works. Sorry for tagging you